### PR TITLE
Port motion-planning-api to ROS2

### PR DESCRIPTION
**Pull request by facontidavide**
### Description

Ported one of the tutorials to ROS2

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)

### CODE REVIEW

**Comment by vatanaksoytezer** [comment]
@facontidavide do you mind running pre-commit for formatting?

PS: If you can `pre-commit install`. It would automatically do formatting for you in every git-commit from now on.

File: doc/motion_planning_api/motion_planning_api_tutorial.rst [hunk 1]
```diff
@@ -16,20 +11,20 @@ Getting Started
 ---------------
 If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
 
+This tutoral has made use of **xterm** and a simple prompter to help the user progress through each demo step.
+To install xterm please run the following command: ::
+
+   sudo apt-get install -y xterm
```
Comment (tylerjw) [inline doc/motion_planning_api/motion_planning_api_tutorial.rst:17]: This is no longer necessary, please convert this to use the rvizvisualtools prompt?

**Comment by vatanaksoytezer** [comment]
@facontidavide can we get a rebase on this one?

**Review by tylerjw — COMMENTED** [review]
This also needs rebased.

**Comment by vatanaksoytezer** [comment]
Closing since #247 has been merged.

**Pull request closed without merging**